pandas/core/categorical.py

```diff
@@ -356,8 +357,26 @@ def itemsize(self):
         """ return the size of a single category """
         return self.categories.itemsize
 
-    def reshape(self, new_shape, **kwargs):
-        """ compat with .reshape """
+    def reshape(self, new_shape, *args, **kwargs):
+        """
+        An ndarray-compatible method that returns
+        `self` because categorical instances cannot
+        actually be reshaped.
+        """
+        # while the 'new_shape' parameter has no effect,
+        # we should still enforce valid shape parameters
+        shape = np.asarray(new_shape)
```

Comment:
Actually, looking at it again, what do you mean by "defer to `numpy`"?  All the code does is return `self` as it always did.  My changes are just preventing abuse of `new_shape`.